docs/versioning.md

```diff
@@ -0,0 +1,114 @@
+# Assembly Versioning
+
+This document lists our guidelines for incrementing the assembly versions for *WinRT.Runtime.dll* and *Microsoft.Windows.SDK.NET.dll*, which are the runtime and SDK projection assemblies generated by C#/WinRT. If we increment [AssemblyVersion](https://docs.microsoft.com/dotnet/standard/assembly/versioning) for either of these assemblies, application developers may have to upgrade their SDK to the newest version and re-compile their applications. This may become painful for developers as we make frequent updates to the .NET 5.0 SDK through servicing and feature releases, so the recommendation from the .NET team is to avoid making changes to `AssemblyVersion` when possible aside from major releases.
```

Comment:
This document lists the guidelines we will be following for ...?  Do you think that might be better?